src/components/RestaurantCard/index.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import Style from "./styles.module.css";
+import { FiPackage } from "react-icons/fi";
+import { FaMotorcycle } from "react-icons/fa";
+
+const RestaurantCard = ({
+  restaurantItem = {
```

Comment:
If RestaurantCard receives just one prop, won't it be more convenient to spread its inner keys?